### ISSUE TITLE
Add a "JUMP" step in SSA trace

### ISSUE THREAD

**Issue by rafaelsamenezes** [esbmc/esbmc#1939]
In ESBMC, our counterexample is generated using the SSA trace. However, it can be challenging to track the program flow when only the assignments are available. We could make the CE more complete by adding a "JUMP" step in the trace which will track all jump instructions (e.g., function calls, if statements, return from function). Of course, it will be hidden by default and it will have no impact in the analysis.

**Comment by fbrausse** [comment]
Sounds good. I also noticed that some traces are extremely hard to follow because the slicer is removing large chunks of assignments inbetween and because the printed trace doesn't provide full stack traces. I'm fine with this proposal, but suggest to, as usual, run it over sv-comp just to be sure we're not introducing too much overhead.
